README.md

```diff
@@ -34,7 +33,6 @@ things.
 Getting Started
```

Comment:
Can't tag the exact line (old 28, new 27) in the review

> You have found a bug or you have an idea for a cool new feature? Contributing

into (add comma after bug)

> You have found a bug, or you have an idea for a cool new feature? Contributing